test/Tester/CancellationTests/GrainCancellationTokenTests.cs

```diff
@@ -21,8 +21,6 @@ protected override TestCluster CreateTestCluster()
         }
 
         [Theory, TestCategory("BVT"), TestCategory("Functional"), TestCategory("Cancellation")]
-        [InlineData(0)]
-        [InlineData(10)]
         [InlineData(300)]
```

Comment:
I would think it's better to comment them instead of removing those lines... otherwise it's not clear why it's a Theory at all (and also it's easier to uncomment and start troubleshooting the underlying issue, in case someone wants to jump in)